src/Chapter1.hs

```diff
@@ -417,6 +429,7 @@ task is to specify the type of this function.
 49
 -}
 
+squareSum :: Num a => a -> a -> a
```

Comment:
Nice work on specifying the most general type for this function! In this task, we don't require to specify the polymorphic type, as we haven't explained it yet. But you are already one step ahead! :top: